app/components/funding_widget_component.html.erb

```diff
@@ -30,47 +30,33 @@
           <div class="call-to-action__text">
             <h3><%= sub_head %></h3>
             <div>
-              <p>
-                A scholarship or bursary isn't available for your chosen subject, but you can still get a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a> if you're eligible (non-UK citizens without indefinite leave to remain in the UK are unlikely to be eligible).
-              </p>
-
-              <p>
-                You might be able to get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> and <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.
-              </p>
-            </div>
-          </div>
-          <div>
-            <h3>Next steps</h3>
-            <div>
-              <p>Use the <a href="https://www.gov.uk/student-finance-calculator">student finance calculator</a> to find out what tuition fee and maintenance loan you could get.<br></p>
-              <p>Explore <a href="/train-to-be-a-teacher">how to train to be a teacher</a>.</p>
+              <p>For this option, a scholarship or bursary is not available. You can still get a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a> if you're eligible.</p>
+              <p>You might be able to get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> or <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <%= render Content::InsetTextComponent.new(
+                header: "Non-UK citizens:",
+                text: "You're unlikely to get help funding your training unless you have permission to <a href=\"https://www.gov.uk/browse/visas-immigration/settle-in-the-uk\">live permanently in the UK</a>. Find out about <a href=\"/non-uk-teachers/fees-and-funding-for-non-uk-trainees\">funding for non-UK citizens</a>.",
+                color: "purple-white"
+              ) %>
             </div>
           </div>
         <% else %>
           <div class="call-to-action__text">
             <h3><%= sub_head %></h3>
             <div>
               <%= helpers.safe_format(funding_results) %>
-
               <p>You can receive this alongside a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a>.</p>
-              <p>You could also get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> and <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
+              <p>You could also get extra funding support <a href="/funding-and-support/if-youre-a-parent-or-carer">if you're a parent or carer</a> or <a href="/funding-and-support/if-youre-disabled">if you're disabled</a>.</p>
```

Comment:
Hey @martyn-w any idea why the paragraph breaks aren't picked up? I've displays without a space between them and I'm not sure why
![Screenshot 2024-08-09 at 15 44 37](https://github.com/user-attachments/assets/f9b7bc12-9ec7-4a72-810d-2bd48efd42da)